src/bundle/views/BundleDetail.tsx

```diff
@@ -0,0 +1,443 @@
+import { useMutation } from '@apollo/react-hooks';
+import { get, isEmpty } from 'lodash-es';
+import React, { FunctionComponent, ReactText, useEffect, useState } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { withRouter } from 'react-router';
+
+import {
+	Avatar,
+	BlockHeading,
+	Button,
+	ButtonToolbar,
+	Column,
+	Container,
+	DropdownButton,
+	DropdownContent,
+	Flex,
+	FlexItem,
+	Grid,
+	Header,
```

Comment:
Wordt niet gebruikt

Reply:
fixed in latest bundle branch: feature/AVO2-374-duplicate-bundle-and-collection